pylint/checkers/python3.py

```diff
@@ -0,0 +1,40 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+"""This is the remnant of the python3 checker. It was removed because
+the transition from python 2 to python3 is behind us, but some checks
+are still useful in python3 after all.
+See https://github.com/PyCQA/pylint/issues/5025
+"""
+
+from astroid import nodes
+
+from pylint import checkers, interfaces
+from pylint.checkers import utils
+from pylint.lint import PyLinter
+
+
+class Python3Checker(checkers.BaseChecker):
+
+    __implements__ = interfaces.IAstroidChecker
+    enabled = True
```

Comment:
```suggestion
    enabled = False
```
If we want to make this check optional we could do this.